examples/main.workflow

```diff
@@ -1,13 +1,13 @@
-workflow "Update" {
+workflow "Update " {
```

Comment:
Nit: Why the space after "Update"?

Reply:
Mistake, I'll fix, thanks for noticing it.